modules/ROOT/pages/supported_versions.adoc

```diff
@@ -3,9 +3,10 @@ include::_partials/attributes.adoc[]
 // end {partialsdir}/attributes.adoc[]
 = Supported Versions
 
-Generally speaking, AeroGear Mobile Services aim to support the latest versions of tools and technologies. As of Feb 2019, the following versions are tested and supported:
+Generally speaking, AeroGear Mobile Services aim to support the latest versions of tools and technologies.
+As of July 2019, the following versions are tested and supported:
 
-* Cordova: version 8
-* Nodejs: version 8 and 10
-* Android SDK: API level 27
-* XCode: version 10.1
\ No newline at end of file
+* Cordova: version 9
+* Nodejs: version 10 and 12 LTS
+* Android SDK: API level 29
```

Comment:
Should this be 28? From what I've looked, 29 was not released yet.

Reply:
I remember @danielpassos trying our stuff on top of the beta release. 
I wanted to save another bump but we can downgrade to 28

Reply:
Well, Android 29 still in [preview version](https://developer.android.com/preview)

Reply:
I think it's safer to state "Android SDK: API level 28". Does any know what version(s) did we test this on/plan to test on?

Reply:
Currently our integration tests run against API level 28, also device we are/will be testing on manually is 28.

Reply:
Agree! 28 is better for now

Reply:
Great - I'll make the udpate :+1: